engines/stark/gfx/openglsactor.cpp

```diff
@@ -112,6 +116,34 @@ void OpenGLSActorRenderer::render(const Math::Vector3d &position, float directio
 	}
 
 	_shader->unbind();
+
+	if (castsShadow && StarkSettings->getBoolSetting(Settings::kShadow)) {
+		glEnable(GL_BLEND);
+		glEnable(GL_STENCIL_TEST);
+
+		_shadowShader->enableVertexAttribute("position1", _faceVBO, 3, GL_FLOAT, GL_FALSE, 14 * sizeof(float), 0);
```

Comment:
Although you're just emulating code that is already written, I would like to remark two things:
1. Vertex attributes should only be set once, when the corresponding VBO object is created, rather than for every render operation.
2. Both the stride (second to last parameter) and the offset (last parameter) should be defined in terms of multiples of floats.